wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+
+Do note that depending on the player's settings, the spectators' account name in a list format will be shown at the player's screen on the top-left of the screen in white text.
+
+A format and example is shown below for the player:-
+
+```
+Format :-
+Spectator list (#):
+{accountNameInNewlines}
+
+Example:-
+Spectator list (2):
+deadbeat
+ztrot
+```
+
+A white, hovering text from right-to-left will be shown on top when watching a spectator's replay.
+
+The format and example is shown below:-
+
+```
+Format : SPECTATOR MODE - Watching {AccountName} play {ArtistName} - {BeatmapsetName} [{Difficulty}]
+Example: SPECTATOR MODE - Watching peppy play Peter Lambert - osu!tutorial [Gameplay Basics]
+```
+
+### Auto
+
+![An Auto replay when connected to Bancho][Replay Auto on]
+
+_A typical Auto replay_
+
+The Auto game modifier plays the difficulty/beatmap.
+It does not save internally but **can be exported**.
+Opening an exported file of Auto play will put the play into the local leaderboard with a unique Auto game modifier used.
+
+Do note that the **UNRANKED** notice cannot be removed at all.
+
+#### Preview Gameplay
+
+When the `Preview Gameplay` button in the _Options_ sidebar was clicked, Auto will **randomly take a difficulty/beatmap** based on **currently selected game mode (set in Solo's Song Selection screen)** and **a random timespan to start**.
+
+In this type of replay, Auto may randomly fail on some hitobjects to show how the skins elements looks like.
+If the `Preview Gameplay` button was not clicked again to randomise another difficulty/beatmap and timespan when the song ended, result screen will be shown for this play and the _Options_ sidebar will be closed.
+
+The _Options_ sidebar cannot be reopened once left during play/in result screen.
+
+### Server
+
+[A server replay example][Replay server]
+
+_A server replay_
+
+Server replays are reserved for top 50 play in the global leaderboard of a difficulty/beatmap.
+The replay will be saved server-side.
+It can be exported, or downloaded and watched immediately by request.
+
+When a server-side replay was watched (excluding exported replay), the player of the replay's profile will received an increment of one(1) in the replay counter.
+If a new top 50 global record was added, the previous holder for \#50 position's server-side replay will be removed.
+
+To watch a server replay, an osu!account must be logged in-game connected to Bancho server online.
+At the _Song Selection_ screen in _Solo_, switch the leaderboard to _Global leaderboard_ and click on the preferred player at the leaderboard to watch the replay.
+Press on the _Watch replay_ button and osu! will download the replay file from the Bancho server as shown in the left of the image.
+The replay only plays once and will be deleted after the replay ends/quit early.
+
+## Trivia
+
+### General
+
+Playback speed does not affect the `REPLAY MODE/SPECTATOR MODE` notice.
```

Comment:
`Playback speed does not affect the `REPLAY MODE/SPECTATOR MODE` notice scrolling speed.`